opossum-tool/OpossumUI
# Needs review checkbox

### Summary of changes

In this branch we are adding a feature to signal that an attribution needs review.

### Context and reason for change

In order to improve the workflow of a typical QA process, we add this feature to provide a way for an OpossumUI user to signal to QA that an attribution needs special attention.

### How can the changes be tested

Open a file in OpossumUI and use the checkbox for a manual attribution. Check also that the statistics popup is updated accordingly.
